app/scripts/controllers/metametrics.js

```diff
@@ -132,7 +133,11 @@ export default class MetaMetricsController {
     } else if (participateInMetaMetrics === false) {
       metaMetricsId = null;
     }
-    this.store.updateState({ participateInMetaMetrics, metaMetricsId });
+    this.store.updateState({
+      participateInMetaMetrics,
+      metaMetricsId,
+      metaMetricsParticipationSet: true,
```

Comment:
elsewhere we check if 'participateInMetaMetrics' is null, which is its default value, and once a user selects one of the two choices it becomes true or false. I do like the consistency of having a literal value in state, and it's definitely more explicit. I'm wondering, though, if we could make it slightly more explicit with "metaMetricsOnboardingCompleted" ... as we *can* check if participateInMetaMetrics has been set by comparing it to true/false.